ecommerce/android/Getflix/app/src/main/java/com/example/getflix/models/ProductModel.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.getflix.models
+
+data class ProductModel(
+    var id: Double,
+    var name: String,
+    var category: String,
+    var price: Float,
+    var brandId: String,
+    var description: String
+)
```

Comment:
I think it is better to create product model as a Parcelable data class in order to be handy in the use `ProductModel`  in the models of Subcategory and Category.

```suggestion
package com.example.getflix.models

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize


@Parcelize
data class ProductModel(
    var id: Double,
    var name: String,
    var category: String,
    var price: Float,
    var brandId: String,
    var description: String
) : Parcelable
```